nixos/modules/programs/kdeconnect.nix

```diff
@@ -0,0 +1,23 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  meta.maintainers = [ maintainers.eliasp ];
+
+  options = {
+    programs.kdeconnect = {
+      enable = mkOption {
+        default = false;
+        type = types.bool;
+        description = "Enable KDE Connect and configure the firewall for it.";
+      };
+    };
+  };
+
+  config = mkIf config.programs.kdeconnect.enable {
+    environment.systemPackages = [ pkgs.kdeconnect ];
```

Comment:
Would be a systemd.user.service useful here? https://github.com/NixOS/nixpkgs/pull/51379

Reply:
I don't believe you need that - at least in my experience it seems to startup automatically (in KDE at least). I think there's a dbus service?

Reply:
I'm seeing a xdg autostart and the dbus service `org.kde.kdeconnect.service`.